lib/improver/nowcasting/optical_flow.py

```diff
@@ -932,13 +945,13 @@ def process(self, cube1, cube2, boxsize=30):
         t_coord = cube2.coord("time")
 
         ucube = iris.cube.Cube(
-            ucomp, long_name="advection_velocity_x", units="m s-1",
-            dim_coords_and_dims=[(y_coord, 0), (x_coord, 1)])
+            ucomp, long_name="eastward_precipitation_advection",
```

Comment:
The cube names were decided at the tech lead level - @benfitzpatrick suggested this which is why I changed it.

Reply:
It might be best if this was `eastward_precipitation_velocity`, although this only truly makes sense, if the velocity is genuinely eastward, but actually, due to the grid being used, it's unlikely to be actually eastward. `precipitation_x_velocity` would be an alternate.

Reply:
After talking to @benfitzpatrick, `precipitation_advection_x_velocity` and `precipitation_advection_y_velocity` might be slightly preferable given that we're talking about grid east and grid north, rather than true east and true north.